src/test/java/com/google/graphgeckos/dashboard/api/DashboardControllerTest.java

```diff
@@ -0,0 +1,167 @@
+package com.google.graphgeckos.dashboard.api;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.google.cloud.Timestamp;
+import com.google.graphgeckos.dashboard.datatypes.*;
```

Comment:
please expand this [wildcard import](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
Done!